tests/bugzilla/Bugzilla1251.java

```diff
@@ -37,6 +37,19 @@
  * Testlet for reproducing Bugzilla reported bugs.
  *
  * Reported-by: slobo@matavnet.hu
+ * 
+ * jkesselm May2023: CONFIRMED OPPOSITE. Despite claim from user, identity transformer reports FAIL (throws exception), 
+ * real transformer reports PASS (exception caught and handled as intended).
+ * 
+ * In real transformer, ElemCopy catches the SaxException, throws it wrapped in TransformerException;
+ * TransformerImpl catches that, sees that it has an m_serializationHandler. and passes it off to
+ * that. Since that reports checkPass, test is considered good.
+ * 
+ * In TransformerIdentityImpl.startElement(), there is no equivalent catch-and-handle for exceptions.
+ * 
+ * RECOMMENDATION: SANITY CHECK. User appears to have misstated the problem. If so -- if the description is
```

Comment:
-- -> —

Reply:
Sorry, I write in 8-bit ASCII unless there is good reason not to. Suggestion declined.

Reply:
As JEP 400 (https://openjdk.org/jeps/400) confirms, UTF-8 is the way to go. See https://bugs.openjdk.org/browse/JDK-8301971

Reply:
Mumble. I honestly think this is Not An Improvement. We're typing, not typesetting. And -- is pervasive throughout this codebase.

Since folks seem to care, I'm willing to let this instance be changed. 

I suppose I could bind a key to M-dash and try to get in the habit of using it in the future. 

I am *not* volunteering to do a global grovel through the code and change the 300+ other instances; if it's important to you, feel free to do so and I'll reluctantly agree with the pull requests.

Reply:
Change pushed.